crates/api/src/callable.rs

```diff
@@ -157,7 +157,9 @@ impl WrappedCallable for WasmtimeFn {
             .context()
             .compiler()
             .get_published_trampoline(body, &signature, value_size)
-            .map_err(|_| HostRef::new(Trap::fake()))?; //was ActionError::Setup)?;
+            .map_err(|e| -> Trap {
+                HostRef::new(TrapInfo::new(format!("trampoline error: {:?}", e))).into()
```

Comment:
I think this may be an accidental syntax error?

Reply:
Not really, making it more readable though.